hello-world/src/test/java/org/acme/schooltimetabling/solver/TimetableEnvironmentTest.java

```diff
@@ -0,0 +1,45 @@
+package org.acme.schooltimetabling.solver;
+
+import static org.acme.schooltimetabling.TimetableApp.generateDemoData;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+
+import ai.timefold.solver.core.api.solver.Solver;
+import ai.timefold.solver.core.api.solver.SolverFactory;
+import ai.timefold.solver.core.config.solver.EnvironmentMode;
+import ai.timefold.solver.core.config.solver.SolverConfig;
+
+import org.acme.schooltimetabling.TimetableApp;
+import org.acme.schooltimetabling.domain.Lesson;
+import org.acme.schooltimetabling.domain.Timetable;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+@EnabledIfSystemProperty(named = "slowly", matches = "true")
+class TimetableEnvironmentTest {
+
+    @Test
+    void solve() {
+        solve(EnvironmentMode.FULL_ASSERT);
+        solve(EnvironmentMode.FAST_ASSERT);
+    }
```

Comment:
I like this. Simple, no boilerplate, no profiles.
Do you mind propagating this to the rest of the tests?
(Spring Boot can do it differently, if this causes trouble.)